es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,258 +1,258 @@
 **1. Machine Learning tips and tricks cheatsheet**
 
-&#10230;
+&#10230; Hoja de referencia y ayudas de aprendizaje automático
 
 <br>
 
 **2. Classification metrics**
 
-&#10230;
+&#10230; Métricas de clasificación
 
 <br>
 
 **3. In a context of a binary classification, here are the main metrics that are important to track in order to assess the performance of the model.**
 
-&#10230;
+&#10230; En el contexto de una clasificación binaría, estas son las principales métricas que se tienen que seguir para evaluar el rendimiento del modelo
 
 <br>
 
 **4. Confusion matrix ― The confusion matrix is used to have a more complete picture when assessing the performance of a model. It is defined as follows:**
 
-&#10230;
+&#10230; Matriz de confusión ― La matriz de confusión es usada para tener un mejor panorama cuando se evalua el rendimiento del modelo. Se define como:
 
 <br>
 
 **5. [Predicted class, Actual class]**
 
-&#10230;
+&#10230; [Clase Predicha, Clase Real]
 
 <br>
 
 **6. Main metrics ― The following metrics are commonly used to assess the performance of classification models:**
 
-&#10230;
-
+&#10230; Métricas principales ― Las siguientes métricas son comúnmente usadas para evaluar el desempeño de modelos clasificatorios
```

Comment:
Could you replace every "desempeño" in the text with _rendimiento_ (there are a few)? It is just to be consistent with other translations.